docs/_test.md

```diff
@@ -36,7 +36,7 @@ class X:
 This is probably not implemented in the best way, but for now it works:
 
 ```python
-import ^[info](strawberry)
+import [info](strawberry)
```

Comment:
@sairevanth0007 was this failing? 

the `^` was on purpose :D

Reply:
yes i had got this in failing errors.

Reply:
uhm, gotcha, I'll take a better look later tonight or tomorrow